src/main/kotlin/eu/europa/ec/eudi/pidissuer/adapter/out/pid/GetPidDataFromAuthServer.kt

```diff
@@ -35,10 +35,20 @@ private val log = LoggerFactory.getLogger(GetPidDataFromAuthServer::class.java)
 
 class GetPidDataFromAuthServer(
     private val issuerCountry: IsoCountry,
+    private val issuingJurisdiction: IsoCountrySubdivision?,
+    private val generateDocumentNumber: suspend () -> DocumentNumber,
```

Comment:
No need to inject `generateDocumentNumber` and `generarateAdministrativeNumber`. That's an internal concern of `GetPidDataFromAuthServer`.